copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>All links to docs should use the latest version: <a href="https://pdal.io/en/latest" target="_blank">https://pdal.io/en/latest/</a> or at least the same version. You have a mix of 2.6.0 and 2.7.0 in this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Thanks. I've updated these in the revised notebook.</p>